src/Meziantou.Analyzer/Internals/CultureSensitiveFormattingContext.cs

```diff
@@ -295,7 +295,7 @@ private static bool IsInvariantDateTimeFormat(IOperation? valueOperation)
 
     private static bool IsInvariantTimeSpanFormat(IOperation? valueOperation)
     {
-        return valueOperation is { ConstantValue: { HasValue: true, Value: "c" or "C" } };
+        return valueOperation == null || valueOperation is { ConstantValue: { HasValue: true, Value: null or "" or "c" or "C" } };
```

Comment:
You're amazing! That was probably the fastest bug fix in open source software ever! 🏆

I'd like to draw your attention to the fact, that both `"t"` and `"T"` are also acceptable standard format strings with [the same effect as `"c"`](https://learn.microsoft.com/en-us/dotnet/standard/base-types/standard-timespan-format-strings#the-constant-c-format-specifier), so you might want to add those as well:

![image](https://github.com/meziantou/Meziantou.Analyzer/assets/164360/847d25ac-b7b4-43f9-9947-e8c77e575e0b)


Thanks for delivering a great piece of software with even greater support!

Reply:
Added in https://github.com/meziantou/Meziantou.Analyzer/commit/715447944a8469484cd1b02e64301bd7e2f5bda7